gdl_hyperopt_template.py

```diff
@@ -13,12 +13,12 @@
 import pprint
```

Comment:
Should we remove this script and get it back in old versions of GDL when ready to reimplement? #243 
Not sure it's relevant to keep it and maintain it for now.

Reply:
@victorlazio109 is that okay with you? With hydra, it may be cool to go with optuna for hyperparameter search #243

Reply:
EDIT: No need to remove gdl_hyperopt_template. It will be removed once #278 is merged.